src/components/DailyHoroscope.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import "../App.css";
+import { PuffLoader } from "react-spinners";
+function DailyHoroscope({ sun_sign, signs, date }) {
+  const [sunSignHoro, setSunSignHoro] = useState([]);
+  const [currentDay, setCurrentDay] = useState("today"); // intializes today as the first state
+  const getHoroscopeData = () => {
+    const urlConst = date.map( // construct url to retrieve data for days in date array
+      (d) => `https://aztro.sameerkumar.website/?sign=${sun_sign}&day=${d}`
+    );
+    Promise.all(
+      urlConst.map((url) =>
+        fetch(url, { method: "POST" }).then((data) => data.json())
+      )
+    )
+      .then((data) => { // structure data to mange days with currentDay states
+        const signHoroDataStructure = data.map((d, i) => ({
+          date: date[i],
+          data: d,
+        }));
+        setSunSignHoro(signHoroDataStructure);
```

Comment:
This calls are structured in an array of objects[{date:"today", data:"today's data"}]